<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab3</class>
 <widget class="QWidget" name="Lab3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab3</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Отсечение прямой и области методом Коэна-Сазерленда</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="areaButton">
      <property name="text">
       <string>Область</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="countSideLine"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="windowXLEdit">
      <property name="text">
       <string>200</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="windowYLEdit">
      <property name="text">
       <string>200</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="windowXREdit">
      <property name="text">
       <string>800</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="windowYREdit">
      <property name="text">
       <string>800</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QPushButton" name="windowButton">
      <property name="text">
       <string>Окно</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Заливка узором</string>
   </property>
   <widget class="QPushButton" name="paintButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Заливка</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
